extensions/wikia/PageHeader/PageHeaderController.class.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Wikia\PageHeader;
+
+
+class PageHeaderController extends \WikiaController {
+
+	/** @var  ActionButton */
+	private $actionButton;
+
+	public function init() {
+		$this->actionButton = $button = new ActionButton( \RequestContext::getMain() );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$button" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)